src/Model/Configuration.py

```diff
@@ -0,0 +1,121 @@
+
+import platform
+import ctypes
+import sqlite3
+import os
+import functools
+
+from src.Model.Singleton import Singleton
+
+
+def set_up_hidden_dir():
+    """
+    Set up the hidden directory
+    """
+    directory_name = '.OnkoDICOM'
+
+    # Set USER_HIDDEN and USER_HOME env variable
+    os.environ['USER_HOME'] = os.path.expanduser("~")
+    if platform.system() == 'Windows':
+        path = os.environ.get('USER_HOME') + "\\" + directory_name + "\\"
+    else:
+        path = os.environ.get('USER_HOME') + "/" + directory_name + "/"
+    os.environ['USER_HIDDEN'] = path
```

Comment:
This might clash with another environment variable given it consist of user and a generic word.  Suggest USER_ONKODICOM_HIDDEN or something similar

Reply:
Yes. We will change this and put in another PR.